### ISSUE TITLE
Text (markdown) with link to file cannot be opened

### ISSUE THREAD

**Issue by XueSheng-GIT** [nextcloud/android#7192]
### Steps to reproduce
1. Create a new Textfile via nexcloud 20 webif (text.md).
2. Enter some Text, mark it and add a link to a file (new feature on nextcloud 20).
3. Open that file (text.md) on nextcloud android.
4. File will not open (opening takes forever).

### Expected behaviour
- Text file should just open and be shown.

### Actual behaviour
- Opening text file takes forever.

### Environment data
Android version: Android 10.1

Device model: Huawei Mate20pro

Stock or customized system: Stock EMUI 10

Nextcloud app version: 3.14 RC1

Nextcloud server version: 20.1

Reverse proxy: YES (NGINX)

**Comment by tobiasKaminsky** [comment]
@juliushaertl do you have an idea?

**Comment by juliushaertl** [comment]
Hmm, that is indeed something we haven't considered in https://github.com/nextcloud/text/pull/879

I've opened https://github.com/nextcloud/text/issues/1170 to discuss this at the text app

**Comment by lorenzo-w** [comment]
You labelled this as an _enhancement_? Are you kidding? I cannot open my notes. This is a **bug** that needs fixing.

**Comment by XueSheng-GIT** [comment]
Just want to add another view to this topic. I started using collectives and had to realize that I acually can't use them with the android app because these collectives contain alot of links between files. Thus I can't open the collectives on android and can't really create them because links cannot be created.

**Comment by rotdrop** [comment]
This seemingly is still an issue more than one year later.

**Comment by tobiasKaminsky** [comment]
Yes, it must first be implemented on server side, see https://github.com/nextcloud/text/issues/1170
